src/Sulu/Bundle/DocumentManagerBundle/Resources/config/core.xml

```diff
@@ -7,7 +7,7 @@
         <!-- Core -->
         <service id="sulu_document_manager.event_dispatcher.debug" class="Sulu\Component\DocumentManager\EventDispatcher\DebugEventDispatcher" public="false">
             <argument type="service" id="debug.stopwatch" />
-            <argument type="service" id="logger" />
+            <argument type="service" id="logger" on-invalid="null" />
```

Comment:
This also doesn't seem to be consistent.

Reply:
@danrot the debug event dispatcher is basically for logging so it would be strange not having it in the constructor there.